### PR TITLE
feat(sounds): play sounds only enabled AP-415

### PR DESCRIPTION
**Pull request by WanderingHogan**
What this PR does 📖
**This is a rebase of davids PR https://github.com/Satellite-im/Core-PWA/pull/1027**
play sounds only when enabled on settings/notifications page

Which issue(s) this PR fixes 🔨

AP-415

Special notes for reviewers 🗒️
when commit jest check failed

Additional comments 🎤

### CODE REVIEW

**Comment by molimauro** [comment]
@baymax-codecare instead of this solution can you implement the one discussed yesterday? Moreover it seems that in this pull are missing the modifications on the `jest-mock-canvas` bug, dont know if this has been already resolved somewhere else

File: libraries/SoundManager/SoundManager.ts [hunk 1]
```diff
@@ -37,8 +42,30 @@ export default class SoundManager {
         preload: true,
       })
     }
+
+    //
+    // initialize 'soundsMuteFlag' property
+    //
```
Comment (stavares843) [inline libraries/SoundManager/SoundManager.ts:48]: do we need those comments without any comments? 🔨

**Comment by baymax-codecare** [comment]
> @baymax-codecare instead of this solution can you implement the one discussed yesterday? Moreover it seems that in this pull are missing the modifications on the `jest-mock-canvas` bug, dont know if this has been already resolved somewhere else

@molimauro  I created new PR for jest-mock-canvas. about new changes we discussed yesterday, I will do it.

**Review by phillsatellite — APPROVED** [review]

File: libraries/SoundManager/SoundManager.ts (hunk 1 above)
Comment (baymax-codecare) [reply libraries/SoundManager/SoundManager.ts]: It is removed, @stavares843

**Comment by baymax-codecare** [comment]
@molimauro  I changed all what you requested. plz check

**Review by molimauro — APPROVED** [review]

**Comment by stavares843** [comment]
https://user-images.githubusercontent.com/29093946/151612110-b9f2ce2b-15f4-4678-8bae-3c86b5ac9ed7.mov


on dev, you can untoggle/toggle properly

on this branch, if you toggle one, toggles them all, same with untoggle

**Comment by WanderingHogan** [comment]
@baymax-codecare change requested

File: libraries/SoundManager/SoundManager.ts [hunk 2]
```diff
@@ -20,14 +20,18 @@ export enum Sounds {
...
 export default class SoundManager {
   sounds: Record<Sounds, Howl>
+
   /**
    * @constructs SoundManager
    */
   constructor(volume: number = 1.0) {
+    //
+    // Init 'sounds' property
+    //
```
Comment (stavares843) [inline libraries/SoundManager/SoundManager.ts:30]: do we need those comments without any comments? 🔨

Comment (baymax-codecare) [reply libraries/SoundManager/SoundManager.ts]: @stavares843 removed comments

**Comment by stavares843** [comment]
@baymax-codecare is missing fixing this

https://github.com/Satellite-im/Core-PWA/pull/1173#issuecomment-1024585976

**Comment by baymax-codecare** [comment]
@stavares843 fixed the toggle issue.
rebased from dev branch after that jest check failed.

**Review by Jekrimo — APPROVED** [review]

**Review by baymax-codecare — COMMENTED** [review]
all looks good

**Pull request merged**